ui/core.js

```diff
@@ -13,6 +13,9 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/base.css
+//>>css.structure: ../themes/base/core.css
```

Comment:
`core.js` and `core.css` are quite different. There's nothing in this JS file that actually depends on any CSS. I'd like to get input from @arschmitz on this, but as far as I can tell, `core.css` should be referenced as `css.structure` by all widgets that need it, but not here.